### PR TITLE
Update compat

### PR DESCRIPTION
**Pull request by omus**

### CODE REVIEW

File: Project.toml [hunk 1]
```diff
@@ -12,11 +12,11 @@ TimeZones = "f269a46b-ccf7-5d73-abea-4c690281aa53"
 UUIDs = "cf7118a7-6976-5b1a-9a39-7adc72f591a4"
 
 [compat]
-AWSCore = ">= 0.3.0"
-Memento = ">= 0.12.0"
+AWSCore = "0.6"
+Memento = "0.12"
```
Comment (iamed2) [inline Project.toml]: Do you know that this is incompatible with Memento `master`?

Comment (omus) [reply Project.toml]: Can you elaborate?

Comment (iamed2) [reply Project.toml]: Just wondering why you didn't include 0.13 here

Comment (omus) [reply Project.toml]: I didn't include it as it hasn't been released yet so there is very slim change it could introduce an incompatibility

Comment (iamed2) [reply Project.toml]: Alright, well we should try to get that to 1.0 soon so we don't have to keep making PRs here even though the code isn't changing.

**Pull request merged**